### PR TITLE
quality selector radio button thing

### PR DESCRIPTION
**Pull request by PickleNik**
image

### CODE REVIEW

File: app/src/main/kotlin/com/hyperion/ui/screen/PlayerScreen.kt [hunk 1]
```diff
@@ -173,21 +173,18 @@ private fun PlayerScreenLoaded(
                 ) {
                     items(viewModel.video!!.streams.filterIsInstance<DomainStream.Video>()) { stream ->
                         Row(
-                            modifier = Modifier.clickable { selectedStream = stream },
+                            modifier = Modifier.fillMaxWidth(1f).clickable { selectedStream = stream },
```
Comment (zt64) [inline app/src/main/kotlin/com/hyperion/ui/screen/PlayerScreen.kt:176]: This should be chained onto multiple new lines

**Review by zt64 — CHANGES_REQUESTED** [review]

File: app/src/main/kotlin/com/hyperion/ui/screen/PlayerScreen.kt (hunk 1 above)
Comment (PickleNik) [reply app/src/main/kotlin/com/hyperion/ui/screen/PlayerScreen.kt]: fixd, forc pushd

**Comment by PickleNik** [comment]
@zt64 @zt64

**Comment by PickleNik** [comment]
shrimpossible

**Pull request closed without merging**